api.md

```diff
@@ -878,6 +884,216 @@ SELECT reorder_chunk('_timescaledb_internal._hyper_1_10_chunk', 'conditions_devi
+'time', timescaledb.compress_segmentby = 'device_id');
+```
+
+## Compress Chunk (Compression) :community_function: [](compress_chunk)
+
+compress_chunk function is used to compress a specific chunk.
+
+The syntax is:
+
+``` sql
+SELECT compress_chunk( '<chunk_name>');
+```
+#### Sample Usage [](compress_chunk)
+In this example we will compress chunk 1_2
+``` sql
+SELECT compress_chunk( '_timescaledb_internal._hyper_1_2_chunk');
+```
+
+#### Required Arguments [](compress_chunk-required-arguments)
+
+|Name|Description|
+|---|---|
+| `chunk_name` | Name of the chunck to be compressed|
+
+## Compress Policy (Compression) :community_function: [](compress_policy)
+Allows you to set a policy based on a chunks age as to when it will be compressed.
+
+The Syntax is:
+
+``` sql
+SELECT add_compress_chunks_policy('<table_name>', '<time interval>'::interval);
+```
+#### Sample Usage [](compress_policy)
+In this example we will compress chunks older than 60 days from the 'cpu' table.
+
+``` sql
+SELECT add_compress_chunks_policy('cpu', '60d'::interval);
+```
+#### Required Arguments [](compress_policy-required-arguments)
+
+|Name|Description|
+|---|---|
+| `table_name` | Name of the the policy will act on.|
+| `time_interval` | Age at which the policy job will compress chunks.|
+
+## Compress Policy Remove (Compression) :community_function: [](compress_policy_remove)
+If you need to suspend or remove the compression policy. To start policy basd compression again
+you will need to re-add the policy.
+
+The Syntax:
+
+``` sql
+SELECT remove_compress_chunks_policy('<table_name>');
+
+```
+#### Sample Usage [](compress_policy_remove)
+In this example we will remove the compression polict from the 'conditions' table:
+``` sql
+SELECT remove_compress_chunks_policy('conditions');
+```
+#### Required Arguments [](compress_policy_remove-required-arguments)
+
+|Name|Description|
+|---|---|
+| `table_name` | Name of the table the policy should be removed from.|
+
+
+## Decompress Chunk (Compression) :community_function: [](decompress_chunk)
+If you need to modify or add data to a chunk (backfill) you will need to decompress
+the chunk first using this process.
+
+>:TIP: Prior to decompressing chunks for the purpose of data backfill or updating you should
+first stop any compression policy that is active on the hypertable you plan to perfrom this 
+operation on.  Once the update and/or back fill is complete simplt turn the policy back on
+and the system will recompress your chucks.
+ 
+The Syntax is:
+
+``` sql
+SELECT decompress_chunk( '<chunk_name>'); 
+
+```
+#### Sample Usage [](decompress_chunk)
+In this example we are decompressing chunk 2_2
+
+``` sql
+SELECT decompress_chunk( '_timescaledb_internal._hyper_2_2_chunk');  
+```
+#### Required Arguments [](decompress_chunk-required-arguments)
+|Name|Description|
+|---|---|
+| `chunk_name` | Name of the chunk to be decompressed.|
+---
+## Move Chunks :enterprise_function: [](move_chunk)
```

Comment:
Whichever it is "move_chunk" or "move_chunks", the inconsistency needs to be resolved in the header, function calls and anchors

Reply:
done